app/javascript/Patient/FamilyManageForm/FamilyFormManager.res

```diff
@@ -0,0 +1,42 @@
+let s = React.string
+
+open Belt
+
+let count = ref(0)
+type state = array<FamilyMemberForm.FamilyMember.t>
+
+type action =
+  | AddFamilyMember
+  | DeleteFamilyMember(FamilyMemberForm.FamilyMember.t)
+
+let reducer = (state, action) =>
+  switch action {
+  | AddFamilyMember => {
+      count := count.contents + 1
+      Belt.Array.concat(state, [FamilyMemberForm.FamilyMember.make(~id=count.contents)])
+    }
+  | DeleteFamilyMember(member) => Js.Array.filter(m => m != member, state)
+  }
+
+let initialState = []
+
+@react.component
+let make = () => {
+  let (state, dispatch) = React.useReducer(reducer, initialState)
+
+  <div className="max-w-3xl mx-auto mt-8 relative">
+    {state
+    ->Belt.Array.map(props => {
+      <FamilyMemberForm props onClick={_mouseEvt => DeleteFamilyMember(props)->dispatch} />
+    })
```

Comment:
Js.Array instead of Belt. 

What Rescript Doc says about it. 
```
Default to using the Js module. Most of the APIs in it are runtime-free and compile down to clean, readable JavaScript, which is our priority.
```
https://rescript-lang.org/docs/manual/latest/api